docs/content/walkthrough.md

```diff
@@ -86,9 +86,41 @@ the proxy is using in-memory storage, which is only suitable for trying out the
 for a short period of time, as you will quickly run out of memory and the storage
 doesn't persist between restarts.
 
+### With Docker
+In order to run Athens localy in our dev machine with Docker (proxy enabled), we need first to create a directory that will store the persitant modules.
```

Comment:
I think this line might be more clear as something like `In order to run the Athens Proxy using docker, we need to first...`.

Reply:
updated it, thanks a lot, it is better to read indeed.